### PR TITLE
Update YashPimple.json

### PR DESCRIPTION
**Pull request by YashPimple**
Update bio

## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [X] My change requires changes to the documentation.
- [X] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6987"></a>

### CODE REVIEW

File: data/YashPimple.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "name": "Yash Pimple",
   "type": "personal",
-  "bio": "Open Source Contributor | Web Developer | Blogger",
-  "tags": ["Open Source", "DevOps", "Kubernetes", "Docker", "React", "Node.js"],
+  "bio": "Open Source Contributor | DevOps | Cloud-Native | Tech-Blogger"
```
Comment (ChinmayMhatre) [inline data/YashPimple.json:4]:
```suggestion
  "bio": "Open Source Contributor | DevOps | Cloud-Native | Tech-Blogger",
```

**Review by ChinmayMhatre — DISMISSED** [review]
Looks Good! just a missing semi colon

File: data/YashPimple.json (hunk 1 above)
Comment (YashPimple) [reply data/YashPimple.json:4]: I have corrected the error made and update the JSON file

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**